pkg/utils/dyn_pprof_unix.go

```diff
@@ -0,0 +1,40 @@
+// +build linux darwin freebsd unix
+// Copyright 2020 PingCAP, Inc. Licensed under Apache-2.0.
+
+package utils
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/pingcap/log"
+	"go.uber.org/zap"
+)
+
+const (
+	startPProfSignal = syscall.SIGUSR1
+)
+
+var (
+	signalChan = make(chan os.Signal, 1)
+)
+
+// StartDynamicPProfListener starts the listener that will enable pprof when received `startPProfSignal`.
+func StartDynamicPProfListener() {
+	signal.Notify(signalChan, startPProfSignal)
+	go onSignalStartPProf(signalChan)
+	log.Debug("dynamic pprof enabled",
+		zap.Stringer("enableBySignal", startPProfSignal),
+		zap.Int("ppid", os.Getppid()),
```

Comment:
why `Getppid()` (which produces the PID of the parent process)?

Reply:
Seems `Getpid()` will give us the pid of the carrier thread of this goroutine at my computer (Linux 5.7.5-arch1-1), which (IMO) might change during execution.

In practice, I use `pkill -10 -P $PPID` to send the signal... Or the pprof won't be started. :(

Any better idea here if possible?

Reply:
Since this might be useless and misleading, maybe just remove it would be a better idea...